database/migrations/create_permission_tables.php.stub

```diff
@@ -78,7 +78,8 @@ class CreatePermissionTables extends Migration
 
             $table->primary(['permission_id', 'role_id']);
 
-            app('cache')->forget(config('permission.cache.key'));
+            $cacheConfig = config('permission.cache');
+            app('cache')->store($cacheConfig['store'])->forget($cacheConfig['key']);
```

Comment:
This will probably work better:
```php
            app('cache')
            ->store(config('permission.cache.store') != 'default' ? config('permission.cache.store') : null)
            ->forget(config('permission.cache.key'));
```

Reply:
much better :)